src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,10 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        int ballQuantity = 3;
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < ballQuantity; i++) {
+            System.out.println(lottery.getRandomBall().toString());
```

Comment:
`toString()` is redundant, it will be called automatically under the hood

Reply:
deleted